README.md

```diff
@@ -81,9 +112,8 @@ Following interfaces are exposed by the Golang library to be consumed by the use
 
 ## Future Support
 
-- Some of the topics that require further discussion are the following:
-    - Do we want to provide functionality for microservices to subscribe for updates in certain tables?
-    - Do we need to provide pagination or streaming support using channels?
+- Do we want to provide functionality for microservices to subscribe for updates in certain tables?
+- Do we need to provide pagination or streaming support using channels?
```

Comment:
Remove pagination from this bullet point? We're already doing that, and probably channels are not needed for any kind of pagination support

Reply:
done